package.json

```diff
@@ -31,5 +31,10 @@
     "babel-cli": "~6.14.0",
     "babel-plugin-add-module-exports": "~0.2.1",
     "babel-preset-es2015": "~6.14.0"
+  },
+  "jspm": {
+    "main": "src/index.js",
```

Comment:
Ah, your right, through npm is doesn't work because you added `/src` to `.npmignore`. 

I disagree on transpilation of external libraries, because when issues surface, like https://github.com/micky2be/superlogin-client/issues/25 I have to dive into libraries, add logging etc, having them pre-transpiled makes that code hard to read. 
Also JSPM's link workflow wouldn't work, making testing this code in other apps a lot harder.